### PR TITLE
Add extra options in aro install step

### PR DESCRIPTION
**Pull request by vishnuchalla**

### CODE REVIEW

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-cluster-density-v2

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-cluster-density-v2

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-cluster-density-v2

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-cluster-density-v2

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-cluster-density-v2

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-cluster-density-v2

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-cluster-density-v2

**Comment by MayXuQQ** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-aro-aro-p1-f30

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml [hunk 1]
```diff
@@ -35,9 +35,13 @@ tests:
   steps:
     cluster_profile: azure-qe
     env:
-      ARO_WORKER_COUNT: "6"
-      ARO_WORKER_VM_SIZE: Standard_D8s_v3
+      ARO_APISERVER_VISIBILITY: Private
+      ARO_INGRESS_VISIBILITY: Private
```
Comment (mgahagan73) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:39]: Please also test this PR with both ARO_APISERVER_VISIBILITY and ARO_INGRESS_VISIBILITY set to public to verify this does not break any existing jobs.

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-cluster-density-v2

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-cluster-density-v2

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-cluster-density-v2

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-aro-aro-p1-f30

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-aro-aro-p1-f30

**Pull request closed without merging**